slim.Dockerfile

```diff
@@ -18,5 +19,9 @@ RUN apk add --no-cache tini
 COPY --from=builder /usr/local/bin/girder /usr/local/bin/girder
 COPY --from=builder /usr/local/lib/python3.7/site-packages/ /usr/local/lib/python3.7/site-packages/
 COPY --from=girder/girder:latest /usr/share/girder/static/ /usr/local/share/girder/static/
+# Add a config file, to bind the server to all network interfaces inside the container
+RUN \
+  mkdir $HOME/.girder && \
+  echo '[global]\nserver.socket_host = "0.0.0.0"\n' > $HOME/.girder/girder.cfg
```

Comment:
I'm generally not a fan of using dotfiles in home directories in docker since that pattern only really exists to avoid being a nuisance to users.  If I `exec` into the container to look for something, I don't want to get tricked by hidden files.

What do you think about using `/etc/girder.cfg` instead? https://github.com/girder/girder/blob/4cb9a149b8645a98f29f6e078a61a472ce803b96/girder/utility/config.py#L41

If I were groping around in the dark looking for default configuration, the first place I'd check is `/etc`.

Reply:
+1 for `/etc/girder.cfg`, it seems like a more appropriate path inside a container.

Reply:
Sounds good.

Reply:
Actually, if a user were to run with a different pid, this config could get broken anyway.  https://docs.docker.com/engine/reference/run/#pid-settings---pid

It's best to not have behavior depend on what user the container was built with.